src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -544,13 +551,15 @@ public IngestResponse ingestFiles(
       throws URISyntaxException, IOException, IngestResponseException {
 
     // the request id we want to send with this payload
-    UUID request = requestId == null ? UUID.randomUUID() : requestId;
+    if (requestId == null || requestId.toString().isEmpty()) {
+      requestId = UUID.randomUUID();
+    }
 
     // We're about to send this request number
-    LOGGER.info("Sending Request UUID - {}", request);
+    LOGGER.info("Sending Request UUID - {}", requestId);
```

Comment:
actually, lets remove this log line. We should have minimum log info or club multiple of them together. more loggers means more space on client machine.